frontend/public/index.html

```diff
@@ -2,19 +2,19 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/lightbulb.ico?v=2" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+<!--    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />-->
```

Comment:
why are these getting commented?

Reply:
When the favicon wasn't updating at first I commented them to see if they were what causing the issue and I forgot about it.  I honestly am not sure what they do.  Do you think I should just delete them?  The href they're referring to is the react logo/favicon

Reply:
should they be changes to our icon?

Reply:
looks like they have both the favicon and png format of the image in the public folder.

Reply:
ooh I see!